komorebi/src/workspace.rs

```diff
@@ -1427,10 +1452,28 @@ impl Workspace {
             }
         }
 
+        for window in self.floating_windows() {
+            if let Ok(details) = (*window).try_into() {
+                vec.push(details);
+            }
+        }
+
         vec
     }
 
     pub fn visible_windows_mut(&mut self) -> Vec<Option<&mut Window>> {
+        //TODO(alex-ds13): This function shouldn't be needed at all. There is no way to have access
```

Comment:
I think you're right and this can be deprecated; the only use of it is in `reap_orphans` and I don't think that needs to be mutable